soh/soh/Enhancements/randomizer/item_location.cpp

```diff
@@ -105,12 +113,12 @@ void ItemLocation::SetAsHinted() {
     hintedAt = true;
 }
 
-RandomizerHintKey ItemLocation::GetHintKey() const {
+std::vector<RandomizerHintKey> ItemLocation::GetHintedBy() const {
     return hintedBy;
 }
```

Comment:
consider returning a const reference here to avoid copying the vector when calling this function.

Reply:
Same thing here as the string, you added the const but not the reference. So unless you intentionally want to copy the vector, you should return `const std::vector<RandomizerHintKey>&` to return a reference to the vector stored in the hint object.